File: app/src/main/java/com/tracker/app/ticket/Ticket.java
package com.tracker.app.ticket;

import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@EqualsAndHashCode
@ToString
public class Ticket {

    private Long id;
    private Long projectId;
    private String title;
    private String info;
    private Type ticketType;
    private Boolean isCompleted;
    private String dateCreated;

    public Ticket(Long id, Long projectId, String title, String info, Type ticketType,
                  Boolean isCompleted, String dateCreated) {
        this.id = id;
        this.projectId = projectId;
        this.title = title;
        this.info = info;
        this.ticketType = ticketType;
        this.isCompleted = isCompleted;
        this.dateCreated = dateCreated;
    }
}

File: client/my-app/src/components/Ticket.js
const Ticket = ({ticket}) => {
    return (
        <div className="ticket-container">
            <h4>{ticket.title}</h4>
            <p>Info: {ticket.info}</p>
            <p>Type: {ticket.type}</p>
            <p>Completed: {ticket.isCompleted}</p>
            <p>Date Created: {ticket.dateCreated}</p>
            <p>Date Completed: {ticket.dateCompleted}</p>
        </div>
    )
}

export default Ticket;